nameko_structlog/nameko_structlog.py

```diff
@@ -40,19 +45,15 @@ def get_dependency(self, worker_ctx):
                     structlog.stdlib.add_logger_name,
                     structlog.stdlib.add_log_level,
                     structlog.stdlib.PositionalArgumentsFormatter(),
-                    structlog.processors.TimeStamper(fmt='iso'),
+                    structlog.processors.TimeStamper(fmt="iso"),
                     structlog.processors.StackInfoRenderer(),
                     structlog.processors.format_exc_info,
                     structlog.processors.UnicodeDecoder(),
                 ]
                 if self.development_mode:
-                    chain.append(
-                        structlog.dev.ConsoleRenderer(colors=use_colors)
-                    )
+                    chain.append(structlog.dev.ConsoleRenderer(colors=use_colors))
                 else:
-                    chain.append(
-                        structlog.processors.JSONRenderer(indent=2, sort_keys=True)
-                    )
+                    chain.extend([structlog.processors.JSONRenderer(sort_keys=self.sort_keys)])
```

Comment:
@racken why don't you just use `append` and you have switched to `extend` ?

Reply:
you are correct @tyler46 , i changed it on latest commit

Reply:
Thanks for the update @racken. I will merge it.